config/routes.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
+require 'sidekiq/web'
+
 Rails.application.routes.draw do
+  authenticate :user do
```

Comment:
suppose `user` shouldn't see this page, but admin

Reply:
I couldn't test it as an admin. I corrected it.